tests/pos-java-interop-separate/annotation-with-inner-class-ref/MyScala_2.scala

```diff
@@ -0,0 +1,6 @@
+object MyScala {
```

Comment:
Could you also extend this test to use the annotation and pass it parameter, to make sure the constructor of the annotation is actually usable from Scala ?